src/wallet/wallet.h

```diff
@@ -200,12 +200,10 @@ typedef std::map<std::string, std::string> mapValue_t;
 
 static inline void ReadOrderPos(int64_t& nOrderPos, mapValue_t& mapValue)
 {
-    if (!mapValue.count("n"))
+    if (!mapValue.count("n") || !ParseInt64(mapValue["n"], &nOrderPos))
```

Comment:
In commit "refactor: Replace use of atoi64 in wallet ReadOrderPos" (87a3f13dfeff735aadb1091183f5269c9ab1a118)

Note: Slightly stricter parsing is done here (no whitespace or trailing characters allowed) but this should be fine since string just comes from calling `i64tostr`. Could potentially log or return an error though if parsing fails, since it would be an indication of corruption.